modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/paths/APIBuilder.path

```diff
@@ -110,6 +110,11 @@
 	<td>//div[@class='dropdown-menu show']//*[@type="checkbox" and @id='autocomplete-exclude-applicationStatus']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>FIELDS_OPTIONS</td>
+	<td>//div[contains(@class,'form-group')]/label[contains(.,'${key_field}')]</td>
+	<td></td>
```

Comment:
you can try to use 'FormViewBuilder#FIELD_LABEL' to replace it, could you take a look at it, thanks!

Reply:
I tried using @Hijky  before but it didn't work, the only pattern and reference I found was to create a path considering the class in common for both and the value of the label, and as I didn't find any, I created this one

Reply:
Could you check it? It can work well on my side. Thanks! `//label[text()='${fieldLabel}']`
![image (1)](https://github.com/liferay-headless/liferay-portal/assets/89900202/ef458b88-e723-43c7-909c-68a7fc5e4881)

Reply:
Yes, you're right, I missed assigning the second fieldLabel value, sorry about that, I corrected it now